scriptmodules/supplementary/bluetooth.sh

```diff
@@ -402,6 +404,8 @@ function gui_bluetooth() {
                     ;;
             esac
         else
+            # restart sixad (if running)
+            service sixad status >/dev/null && service sixad restart
```

Comment:
I'm not exactly sure if restarting the daemon is even necessary. I've tried omitting this line and I can successfully turn off/on & repair my DS3 controller after using the Bluetooth menu.